website/content/docs/enterprise/supported-versions.mdx

```diff
@@ -1,8 +1,8 @@
 ---
 layout: docs
-page_title: Boundary Enterprise supported versions
+page_title: Boundary Enterprise supported version policies
 description: >-
-  The supported versions policy for Boundary Enterprise. Includes information about support periods, control plane and worker compatibility, and Postgres database version support.
+  Learn about Enterprise support policies. Includes information about support periods, control plane and worker compatibility, and Postgres database versions.
```

Comment:
I think both names are acceptable here, just not sure which the tech writers are using? 

```suggestion
  Learn about Enterprise support policies. Includes information about support periods, control plane and worker compatibility, and PostgreSQL database versions.
```

Reply:
Good question, I don't think we have any kind of word list for product names like that. In the docs, I have been trying to use the full version you used in your suggestion `PostgreSQL`, but I missed it here.